app/src/main/res/layout/item_form_spinner_with_header.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/tv_header"
+            style="@style/OpenSansBold14sp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="@color/grey_4F4F4F"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="Name" />
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/tv_required_asterisk"
+            style="@style/OpenSansSemiBold16sp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="4dp"
+            android:text="@string/asterisk"
+            android:textColor="@color/red_asterisk_D31717"
+            app:layout_constraintStart_toEndOf="@id/tv_header"
+            app:layout_constraintTop_toTopOf="@id/tv_header" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Do we need a nested constraint layout here? Check if we can remove that

Reply:
Removed it. As of now there's not use of it was dependent on "isEnable" attribute which is not required in Spinner I guess for this app.